### DISCUSSION TITLE
Feature Ideas

### DISCUSSION THREAD

**Discussion by Fermain** [Fermain/-mollify#9] [Ideas]
Welcome to the Mollify Feature Ideas Discussion Board! This is the perfect place for our vibrant community of educators, content creators, developers, and users to come together and share their thoughts, ideas, and suggestions for new features and improvements in the Mollify Learning Management System (LMS). We're excited to hear your innovative ideas and constructive feedback to help shape the future of Mollify, making it the go-to LMS for creating and managing engaging, interactive, and accessible learning experiences.

Whether you have ideas for enhancing the Markdown and Svelte integration, streamlining the Git workflow, improving AI-powered content generation, or elevating the student experience, we want to hear from you. Your input is invaluable in our pursuit to create an LMS that not only meets the needs of today's educators and learners but also anticipates the demands of the future.

**Comment by Anclagen** [comment]
Some ideas after fiddling for a bit would be
- Quizzes perhaps we create a type to the front-matter that will direct a package to process a page into a multiple choice quiz.
- Optional branding, allow a user to configure their own branding for the navigation moving a powered by mollify logo to the footer.
- Along with use feedback forms and the AI summary of it, we could also add some charts to visually display the data too.
- Something far down the line along with storing bookmarks and login, might be allowing a user to annotate and add notes on a lesson for themselves as well as add comments/discussions for the lesson for all students to see and add to.

**Reply by Fermain** [reply]
I think we should try and squeeze Optional Branding (Whitelabel) into the MVP.

Keen for quizzes but this can be a future goal.

Keen for data vis, although where we store dynamic data in a legally compliant way is an issue.

Very keen for some sort of bookmark or highlight feature. We could use localStorage to begin with until there is a proper persistent DB.

Could you add these as task in the ideas column?
